src/pages/tBTC/Explorer/index.tsx

```diff
@@ -0,0 +1,234 @@
+  RecentDeposit,
+  useFetchRecentDeposits,
+  useFetchTBTCMetrics,
+} from "../../../hooks/tbtc"
+
+const MINTS_TO_DISPLAY = 10
+
+export const ExplorerPage: PageComponent = () => {
+  const { metrics } = useFetchTBTCMetrics()
+  const [latestMints] = useFetchRecentDeposits(MINTS_TO_DISPLAY)
+  const [tvlInUSD, fetchTvl, tvl] = useFetchTvl()
+
+  useEffect(() => {
+    fetchTvl()
+  }, [fetchTvl])
+
+  return (
+    <>
+      <Card
+        as="section"
+        bgImage={tBTCExplorerBg}
+        backgroundRepeat="no-repeat"
+        backgroundPosition="bottom -25px right"
+        backgroundSize="100%"
+      >
+        <Flex as="header" direction="row" alignItems="center">
+          <H5>tBTC TVL</H5>
+          <ButtonLink ml="auto" size="lg" to="/tBTC/mint">
+            Start Minting
+          </ButtonLink>
+        </Flex>
+        <Divider my="8" />
+        <Flex direction={{ base: "column", xl: "row" }} alignItems="center">
+          <H1>{formatFiatCurrencyAmount(tvlInUSD.tBTC, "0,00.00")} USD</H1>
+          <ButtonLink
+            size="lg"
+            variant="outline"
+            href={ExternalHref.tBTCDuneDashboard}
+            isExternal
+            justifySelf="flex-end"
+            ml={{ xl: "auto" }}
+            mt={{ base: 4, xl: "unset" }}
+          >
+            View On Dune Analytics
+          </ButtonLink>
+        </Flex>
+        <Stack
+          mb={{ base: "10", sm: "32", xl: "unset" }}
+          spacing="4"
+          mt="12"
+          direction={{ base: "column", xl: "row" }}
+          justifyContent="space-between"
+        >
+          <MetricBox>
+            <H3>
+              <InlineTokenBalance tokenAmount={tvl.tBTC} />
+            </H3>
+            <BodyMd>
+              <TBTCText />
+            </BodyMd>
+          </MetricBox>
+          <SimpleMetricBox
+            value={formatNumeral(metrics.totalMints, "0,00")}
+            label="Total Mints"
+          />
+          <SimpleMetricBox
+            value={formatNumeral(metrics.totalHolders, "0,00")}
+            label="tBTC Holding Addresses"
+          />
+        </Stack>
+      </Card>
+      <Card mt="4" as="section">
+        <LabelSm as="header">History</LabelSm>
+        <TableContainer mt="6">
+          <Table variant="simple">
+            <Thead color="gray.500">
+              <Tr>
+                <Th paddingLeft={12}>
+                  <LabelXs color="gray.500">
+                    <TBTCText /> amount
+                  </LabelXs>
+                </Th>
+                <Th>
+                  <LabelXs>Wallet Address</LabelXs>
+                </Th>
+                <Th>
+                  <LabelXs>TX Hash</LabelXs>
+                </Th>
+                <Th textAlign={"right"}>
+                  <LabelXs>Timestamp</LabelXs>
+                </Th>
+              </Tr>
+            </Thead>
+            <Tbody>{latestMints.map(renderHistoryRow)}</Tbody>
+          </Table>
+          <BodySm color="gray.500" mt="10">
```

Comment:
```suggestion
          <BodySm color="gray.500" mt="10" mb="2">
```

I think we could add some margin-bottom here so that it looks better on smaller screens when there is a horizontal scroll in the table.

Current implementation:
<img width="265" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/f1f0d2c6-5c05-4fb5-91f0-5feecd515d24">

After adding margin-bottom:
<img width="283" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/0c8f06b2-7d62-47a2-bd77-7275ea885087">

Reply:
7654dfb